packages/flutter_tools/test/general.shard/runner/flutter_command_test.dart

```diff
@@ -42,6 +43,15 @@ void main() {
       when(mockProcessInfo.maxRss).thenReturn(10);
     });
 
+    testUsingContext('FastFlutterCommand updates artifacts', () async {
```

Comment:
Just remove FastFlutterCommand

Reply:
That will trigger more analytics and may slow down time for`flutter test` - is that ok?

Reply:
Yes, I should have never added this hack. The real solution is to make `flutter` start up faster

Reply:
(assuming it is, done - if not I'll revert the last commit in this PR)